.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
```

Comment:
How are `*.md` files managed?

Reply:
md files would comply with these general settings. Would you prefer an override for some of these settings for md files?

Reply:
We probably should do. /cc @SteveLasker 

Here's an example: https://github.com/containerd/containerd/blob/master/docs/.editorconfig